src/Sentry/Exceptions/SentryDebugImage.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Sentry.Extensibility;
+using Sentry.Internal.Extensions;
+
+namespace Sentry
```

Comment:
The file should be on the root of the project (where this namespace represents).